### DISCUSSION TITLE
POSSE Implementation for Rust India content

### DISCUSSION THREAD

**Discussion by me-diru** [rustindia/rustindia.github.io#42] [Ideas]
[POSSE](https://indieweb.org/POSSE) is an abbreviation for Publish (on your) Own Site, [Syndicate](https://indieweb.org/Category:syndication) Elsewhere.  We want to publish the content of Rust India events, posts, updates, etc on a single platform and automate the process of forwarding it to platforms that others are on i.e, Telegram, Discord, Signal, Twitter, Linkedin, etc

This is just the idea and would love to hear your opinion on how to implement it. One possible way is to use [n8n](https://n8n.io/) to automate our workflow with source as our newsletter or website or something neutral

Please share your thoughts and input :D

**Comment by bharattech** [comment]
It sounds great! :+1: I am looking into it, and I'll let you know my thoughts.

**Comment by asdofindia** [comment]
From the time of google reader where we could click a button to share, and then ifttt, and later huginn, etc, I've tried many options and strategies for this. But what worked for me was a manual workflow! This is because of certain reasons:
1) each platform benefits from post crafted for that platform natively, in Twitter and LinkedIn you can tag people, the Twitter character restrictions don't apply for telegram, you can pin a message in telegram, etc etc etc. So using any one thing as source doesn't really scale because we will have to use the lowest common denominator.
2) API based integrations are becoming increasingly harder with platforms closing their APIs
3) There are times when you want to not post certain things in certain places.
4) it is really easy to copy paste the same thing in five platforms.

**Comment by akhildevelops** [comment]
I would suggest using a GitHub repository for single source of truth information. GitHub actions to publish events, posts in various social media channels.

Benefits for the host:
- Need not spend on infra.
- Register events from the community through PRs.
- Easily track all events and posts in markdown files. 
- Update and track markdown files with published status across social media platforms.
- Generate weekly newsletters automatically. 
- Automate processes once PR is approved. 

Benefits for community:
- Need not interact with a new tool.
- Easily connect with the host to share events by submitting PRs.
- Give PR for a published blog / article to share with the community.

The official Rust weekly newsletter follows same idea 💡.
